apple_passkit/lib/src/apple_pk_pass.dart

```diff
@@ -61,13 +61,42 @@ class ApplePkPass {
 }
 
 /// Status after trying to add multiple passes.
-enum PkPassType {
+enum PKPassType {
```

Comment:
Even though the name more closely resembles the iOS name, this is a breaking change. I don't think we should do a bump of the major version for this fix

Reply:
I've already done so